guides/common/attributes-orcharhino.adoc

```diff
@@ -1,25 +1,25 @@
 :KatelloVersion: Katello 3.18
-:ProductVersion: 5.10
-:ProductVersionPrevious: 5.9
+:ProductVersion: 5.11
+:ProductVersionPrevious: 5.10
 :Project: orcharhino
 :ProjectName: orcharhino
 :ProjectNameX: orcharhino
-:ProjectNameXY: orcharhino 5.10
-:ProjectServer: orcharhino server
+:ProjectNameXY: orcharhino 5.11
+:ProjectServer: orcharhino Server
```

Comment:
This looks odd. Shouldn't it be consistent?
```suggestion
:ProjectServer: Orcharhino Server
```

Reply:
As per ATIX Style Guide, we never capitalize orcharhino. However, we defined "orcharhino Server" and "orcharhino Proxy" as a product, same goes for "orcharhino Clients".